app/lib/mammoth/status_origin.rb

```diff
@@ -64,6 +64,12 @@ def find(status_id, acct = nil)
         return results
     end 
 
+    # Delete All Status Origins by username
+    def reset(username)
+        list_key = key(username)
+        redis.keys("#{list_key}*").each { |key| redis.del(key) }
```

Comment:
Should we verify that list_key has a length (not empty) so we don't accidentally remove all entries?

Reply:
oh. `* DEL` that's a good call. 😮